Issue 166: Move error summary to the top

sulthan-ahmed opened the pull request:
As GDS guidelines move error summary to the top
https://github.com/UKHomeOfficeForms/hof/issues/167

zuzak commented:
This commit looks fine to me. That said:

I'm not sure we should be modifying `package-lock.json` without a good reason. Your commit doesn't otherwise rely on changed dependencies. That said, as nobody uses this package as a main app, it looks like that's not as important? (The [docs](https://docs.npmjs.com/files/package-lock.json) say package-lock.json is never published).

If you want to consume these changes, we should create a release `4.0.2`.
You can either do the version bump in `package.json` here, or as a separate commit.

sulthan-ahmed committed:
Issue 166: Move error summary to the top

As GDS guidelines move error summary to the top https://github.com/UKHomeOfficeForms/hof/issues/167
